hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -2143,6 +2166,53 @@ func reconcileControlPlaneOperatorRoleBinding(binding *rbacv1.RoleBinding, role
 	return nil
 }
 
+func reconcileControlPlaneOperatorClusterRole(role *rbacv1.ClusterRole) error {
+	role.Rules = []rbacv1.PolicyRule{
+		{
+			APIGroups: []string{"rbac.authorization.k8s.io"},
+			Resources: []string{
+				"clusterroles",
+				"clusterroles/finalizers",
+				"clusterrolebindings",
+				"clusterrolebindings/finalizers",
+			},
+			Verbs: []string{
+				"get",
+				"list",
+				"watch",
+				"create",
+				"update",
+				"patch",
+				"delete",
+			},
```

Comment:
can we get some more info about why this is necessary? I don't think there's anything occurring here that should require cloud-provider-kubevirt to need to view anything outside of the hosted cluster namespace. All the VM/VMIs and Services are contained within that namespace.

Reply:
The kubevirt-cloud-controller-manager need to get the `Region` and the `Zone` from the infra cluster node, in order to set and use the values also on the Tenant cluster nodes (each VM get the values from the infra node it installed on)
The only way to get those values is actually read them from the nodes labels:
* topology.kubernetes.io/zone
* topology.kubernetes.io/region